README.md

```diff
@@ -41,8 +41,10 @@ cargo run --bin incredible-squaring-avs  start
 cargo run --bin incredible-squaring-avs  start --config-path <PATH>
 ```
 - Simulate slashing
+Run this command. Edit the `operator_1_times_failing` and `operator_2_times_failing` variables in config file based on your preference of they submitting incorrect answer(thereby getting slashed) from 0 to 100.
+
 ```
-cargo run --bin incredible-squaring-avs  start --slash-simulate
+cargo run --bin incredible-squaring-avs  start
```

Comment:
Let's improve the slashing documentation in the readme in another PR.
Here we have two sections with the command `cargo run --bin incredible-squaring-avs start`. We should have just one of those and add a section on slashing, which explains the conditions to be slashed and how it's shown in the example.
Also, some sections are outdated. We should check everything is up-to-date.

Reply:
👍